README.md

```diff
@@ -1 +1,126 @@
-# capston-project-js
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] Kindly remove the `src/module/commentCounter.js` file since it is an empty file and so it is unneeded